modules/apps/headless/headless-builder/headless-builder-impl/src/main/java/com/liferay/headless/builder/internal/resource/BaseHeadlessBuilderResource.java

```diff
@@ -43,4 +45,7 @@ public void setContextHttpServletRequest(
 
 	protected HttpServletRequest contextHttpServletRequest;
 
+	@Context
```

Comment:
is the @Context annotation needed? I think it is injected by our own Vulcan components

Reply:
True!
Also, is not necessary the setter and getter methods for the `contextHttpServletRequest`.
I've deleted the @context and the unused methods.